cmd/eksctl/scale.go

```diff
@@ -0,0 +1,78 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kubicorn/kubicorn/pkg/logger"
+	"github.com/spf13/cobra"
+	"github.com/weaveworks/eksctl/pkg/eks"
+	"github.com/weaveworks/eksctl/pkg/eks/api"
+)
+
+func scaleCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "scale",
+		Short: "Scale resources(s)",
+		Run: func(c *cobra.Command, _ []string) {
+			if err := c.Help(); err != nil {
+				logger.Debug("ignoring error %q", err.Error())
+			}
+		},
+	}
+
+	cmd.AddCommand(scaleNodeGroupCmd())
+
+	return cmd
+}
+
+func scaleNodeGroupCmd() *cobra.Command {
+	cfg := &api.ClusterConfig{}
+
+	cmd := &cobra.Command{
+		Use:   "nodegroup",
```

Comment:
@richardcase would you be interested to add get/create/delete commands also #257? Right now, get will be most important, because user may wish to know how many nodes there are in a given cluster before they scale. They could use kubectl, but we don't have an obvious link from the cluster that kubectl is pointed at, to eksclt, perhaps something to think about (I've opened #258 to continue that discussion).

Reply:
Sure, i'd be happy to do #257 as well. Would you want that before we ship scale?

Reply:
No, I am happy to ship scale first, we will provide docs and it should be
already pretty useful to many people!